### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Tommymarvel**
## Mobile menu

This PR content includes fantastic styling, well-written HTML, and javascript to add functions.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,348 +1,331 @@
...
+        <div class="mobile-menu">
+          <a href="#" class="close-icon" id="closemenu"><img src="/images/Icon.png" alt="X icon">
+            ></a>
+          <ul class="mobile-nav">
+            <li class="nav-item"><a href="#work-section-link" class="nav-link">Portfolio</a></li>
+            <li class="nav-item"><a href="#about-section-link" class="nav-link">About</a></li>
+            <li class="nav-item"><a href="#contact-form-link" class="nav-link">Contact</a></li>
+          </ul>
+        </div>
         <ul class="nav-tags">
```
Comment (tresorsawasawa) [inline index.html:33]: - [ ] Kindly make sure your **Mobile Menu** design sticks to the one from your chosen Figma template as required for the project and then your project will be complete(e.g.: background color, blur effect on the background, font family, opacity,...). Check everything to match your design to the expected one.

| Figma Design | Your Design |
| ------------------- | ------------------- |
| mmf1 | Screenshot from 2022-07-20 00-50-41 |

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Tommymarvel :wave: 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -1,348 +1,331 @@
...
       <a class="logo" href="#">My logo</a>
-      <nav class="nav-bar">
-        <img id="hambuger" src="images/Union.png" alt="hamburger" />
+      <nav class="nav-bars">
+        <div id="hamburger">
+          <i class="fa fa-bars"></i>
+        </div>
+        <div class="mobile-menu">
+          <a href="#" class="close-icon" id="closemenu"><img src="/images/Icon.png" alt="X icon">
+            ></a>
```
Comment (vic778) [inline index.html:26]: - [OPTIONAL] Kindly the close-icon is not showing when the hamburger is pop-up. fix that and let's see your project get approved.
 
Your Design
Screenshot from 2022-07-20 12-49-00

**Review by vic778 — APPROVED** [review]
Hi @Tommymarvel 
Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification

**Pull request merged**